packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -50,6 +51,10 @@ const propTypes = {
    * The maximum width value in px, to be applied to the content container. Used with responsive behavior for actual width.
    */
   contentWidthMax: PropTypes.number,
+  /**
```

Comment:
You can add `@private` here and this will hide the intl prop from being displayed in the docs. Since this prop is added via the `injectIntl` HOC, consumers don't need to be aware/interface with it. Here's [the related PR](https://github.com/cerner/terra-core/pull/2135) that adds support for `@private`

Reply:
@bjankord do we need to log an issue to add this @private key to existing private props?

Reply:
Yeah that would be good to go through and update.